test/test_playbooks/hostgroup.yml

```diff
@@ -0,0 +1,76 @@
+---
+- hosts: fixtures
+  gather_facts: false
+  vars_files:
+    - vars/server.yml
+  tasks:
+  - include: tasks/operating_system.yml
+    vars:
+      operating_system_state: present
+  - include: tasks/ptable.yml
+    vars:
+      ptable_state: present
```

Comment:
I think, we also need `architecture` and `installation_medium` as fixtures here.

Reply:
I have added installation_medium fixture now and there is no architecture module so can't add its fixtures.

Reply:
I think, architecture is safe, because those are always preseeded in foreman.

But does it mean we should have an architecture module?

Reply:
I think they're not preseeded but created by Puppet? But either way, yeah, an architecture module is probably a nice addition. Should be fairly simple.

Reply:
@evgeni: Just for the lolz:

https://github.com/theforeman/foreman/blob/develop/db/seeds.d/060-architectures.rb

Reply:
STFU ;)